src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -194,7 +194,7 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleExit();
             }
 
-            if (commandResult.isTagList()) { // TODO: Remove this in v1.3
```

Comment:
haha wait why did we need to remove this in the first place? i remember it was related to some hack thing?

Reply:
I remove the comment only

Reply:
haha sorry let me rephrase, why was that comment there in the first place?

Reply:
Ohh erm I was really confused w the implementation initially, because I think this is not a good long-term solution but I guess for now, we should just proceed